libcxx/include/__locale

```diff
@@ -516,6 +516,8 @@ protected:
 };
 #endif // _LIBCPP_HAS_WIDE_CHARACTERS
 
+inline _LIBCPP_HIDE_FROM_ABI int __libcpp_isascii(int __c) { return (__c & ~0x7F) == 0; }
```

Comment:
```suggestion
inline _LIBCPP_HIDE_FROM_ABI bool __libcpp_isascii(int __c) { return (__c & ~0x7F) == 0; }
```

Reply:
I could do that, but `isascii` typically returns `int` on platforms that implement it. It's a common extension. WDYT?

Reply:
That looks a lot like a C-ism to me. We're returning a boolean expression, so I don't see the harm in making the return type reflect that. We're also not making it part of the locale base API, so there isn't really a direct connection between `__libcpp_isascii` and the extension.

Reply:
Ok, I am fine with that.